src/components/organisms/WithNavigationBar/WithNavigationBar.tsx

```diff
@@ -48,6 +53,7 @@ export const WithNavigationBar: React.FC<WithNavigationBarProps> = ({
       <div className="navbar-margin">{children}</div>
 
       <Footer />
+      {(venue && <NewProfileModal venue={venue} />) || <Loading />}
```

Comment:
hmm why do we need `<Loading />` here?
```suggestion
      {venue && <NewProfileModal venue={venue} />}
```

Reply:
You are right, I was experimenting with `Loading` in various places and forgot to remove it from here, thanks!
Removed in https://github.com/sparkletown/sparkle/pull/2236/commits/b56f9469f5037f6fecc8399b110cabec4eff4c25